### DISCUSSION TITLE
Get fullName from imported ALB

### DISCUSSION THREAD

**Discussion by jsauter** [aws/aws-cdk#15975] [Q&A]
I am trying to get the load balancer full name to use in a dimension on a cloudwatch metric for an imported ALB.  Using 'fromLookup' or 'fromApplicationListenerAttributes' returns an IApplicationLoadBalancer which doesn't have the loadBalancerFullName property.

Is there a best approach for retrieving a full ApplicationLoadBalancer with CDK, or is there another way to get this property?

**Comment by fmasclef** [comment]
Got the same issue today. I used a SSM parameter to store an ALB fullname. Then, when creating a metric from a stack I imported the ALB, I use:

```js
dimensionsMap: {
  TargetGroup: targetGroup.targetGroupFullName,
  LoadBalancer: ssm.StringParameter.valueForStringParameter(this, `bla bla bla`)
}
```
